res/map/AWBWMapConverter.py

```diff
@@ -0,0 +1,217 @@
+
+import sys
+import math
+import re
+import csv
+
+
+def convertFile(infile,outfile):
+	reader = csv.reader(infile)
+	for line in reader:
+		for num in line:
+			try:
+				num = int(num)
+			except ValueError:
+				num = -1
+			outfile.write(indexToTerrainCode(num))
+		outfile.write("\n")
+	return
+
+def main(names):
+	for name in names:
+		try:
+			infile = open(name)
+			# split the file extension off, and replace it with .map
+			outname = re.sub(r'\..*$','.map',name)
+			try:
+				print(outname)
+				outfile = open(outname, 'w')
+				convertFile(infile,outfile)
+			except IOError:
+				print("File ",outname," cannot be written to.")
+		except IOError:
+			print("File ",name," does not exist.")
+		finally:
+			infile.close()
+			outfile.close()
+
+
+#dictionary text generation
+def ugh():
```

Comment:
This function doesn't appear to be referenced anywhere, and has a pretty nondescriptive name. Is this intermediate code that you forgot to remove?

Reply:
It's code to generate the text of the dictionary. I suppose it's not useful anymore, so I can remove it if it's unwanted.